electives/21270.md

```diff
@@ -0,0 +1,35 @@
+# 21-270: Introduction to Mathematical Finance
+
+| Category | Difficulty |
+| :------: | :--------: |
+|   HW     |     3      |
+|  Exams   |     4      |
+| Quizzes  |     3      |
+
+Disclaimer: Course taken remotely in Spring 2021. 
+
+Course is a math/science elective possible for ECE majors. This course is an introduction to mathematical finance and an introductory course for the computational finance minor/major. This course is primarily using math to price items such as bonds and options. Heavily abstracted, so don't expect to take this course and immediately be able to model the prices of bonds and options, however it is a good starting off point.
```

Comment:
Is there Black Scholes? Are there other major equations you learn about?

Is this useful for people pursuing a career in quant / finance?